Fluentify.Models/Frontend/CreateUserRequest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+
+namespace Fluentify.Models.Frontend
+{
+    public class CreateUserRequest
```

Comment:
From what I can see:
```c#
public string? Nickname { get; set; }
public string? Nickname { get; set; }
public string? Password { get; set; }
``` 
You've made your attributes nullable. This can cause data corruption